### ISSUE TITLE
Presubmit hook should also build affected binaries

### ISSUE THREAD

**Issue by danielmoy-google** [google/kythe#36]
Our presubmit should stop you from checking in a change that breaks a binary but not a test.

**Comment by creachadair** [comment]
I don't think we have any presubmits running tests right now. It would be nice for us to get back to that, though. We've been relying on manually running `bazel test` for some time now.
